pkg/operator/ceph/file/filesystem.go

```diff
@@ -232,3 +255,19 @@ func downFilesystem(context *clusterd.Context, namespace, filesystemName string)
 	logger.Infof("Downed filesystem %s", filesystemName)
 	return nil
 }
+
+// generateDataPoolNames is a function that generates a new name for each pool in the input list suffixed with an appropriate index number, and it returns the renamed pool list
+func generateDataPoolNames(f *Filesystem, spec cephv1.FilesystemSpec) []string {
+	var dataPoolNames []string
+	for i := range spec.DataPools {
+		poolName := fmt.Sprintf("%s-%s%d", f.Name, dataPoolSuffix, i)
+		dataPoolNames = append(dataPoolNames, poolName)
+	}
+	return dataPoolNames
+}
+
+// generateMetaDataPoolNames is a function that generates a name for the MetaDataPool by prefixing the FileSystem's Name to the  constatnt metaDataPoolSuffix
+func generateMetaDataPoolName(f *Filesystem) string {
+	metaDataPoolName := fmt.Sprintf("%s-%s", f.Name, metaDataPoolSuffix)
```

Comment:
nit: you could return on this line instead of assigning to a variable

Reply:
Done.